airflow/providers/google/marketing_platform/hooks/search_ads.py

```diff
@@ -19,73 +19,154 @@
 
 from __future__ import annotations
 
-from typing import Any, Sequence
+from typing import Any
 
 from googleapiclient.discovery import build
 
 from airflow.providers.google.common.hooks.base_google import GoogleBaseHook
 
 
 class GoogleSearchAdsHook(GoogleBaseHook):
```

Comment:
@molcay we shouldn't change this Hook, because the API is still working. It was partially shutdown only for the reporting service. All other services still work and our customers can use this Hook for connection to these services.  Let's create a new Hook for reports API and will deprecate or delete unused methods in the old one.

Reply:
Ohh, I see. I was thinking that since this hook only have methods related to the reporting, and it is shutdown; it might be safe to remove it.